src/validators/psychological.js

```diff
@@ -7,11 +7,11 @@ export default class PsychologicalValidator {
   }
 
```

Comment:
Can we delete `class PsychologicalValidator`?

Reply:
It seems integral to `showQuestion21E` and `hideExistingConditions` in the same file, and those two show up in a lot of different places.

Reply:
Fixed